src/libraries/HooksLib.sol

```diff
@@ -31,6 +40,8 @@ library HooksLib {
         return hookValidationData;
     }
 
+    /// @notice Handles the afterIsValidSignature hook
+    /// @return magicValue the ERC-1271 magic value returned by the hook
```

Comment:
nit: would have this return say smthn like it indicates whether the signature is valid or not, and can override internal logic